go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	github.com/prometheus/common v0.32.1
 	github.com/stretchr/testify v1.7.0
 	golang.org/x/sys v0.0.0-20220114195835-da31bd327af9
-	google.golang.org/grpc v1.41.0
+	google.golang.org/grpc v1.44.0
```

Comment:
Just an FYI: We shouldn't bump these if they haven't been [tested for compatibility](https://github.com/grafana/grafana-plugin-sdk-go/issues/417) - feels like something that could go in another PR

Reply:
@wbrowne removed that.  good to go?